Dockerfile

```diff
@@ -94,7 +94,7 @@ ENV TILEGENERATION_CONFIGFILE=/etc/tilegeneration/config.yaml \
     TILE_CHUNK_SIZE=1 \
     TILE_SERVER_LOGLEVEL=quiet \
     TILE_MAPCACHE_LOGLEVEL=verbose \
-    DEVLOPEMENT=0
+    DEVELOPMENT=0
```

Comment:
It should be removed because it's already defined at line 58 :-)

Reply:
j'ai supprimé